client/webserver/webserver.go

```diff
@@ -1169,6 +1169,20 @@ func folderExists(fp string) bool {
 	return err == nil && stat.IsDir()
 }
 
+// userAppVersion is a template function that returns the user-facing
+// application version, which is the full version without the build metadata. If
+// semVerOnly is true, it returns the version without the pre-release tag. The
+// full version is expected to be in the format
+// "MAJOR.MINOR.PATCH[-PRE-RELEASE][+BUILD-METADATA]". For example,
+// "1.0.4-pre+4ba3fd93b" would return "1.0.4-pre" if semVerOnly is false, and
+// "1.0.4" if semVerOnly is true.
+func userAppVersion(fullVersion string, semVerOnly bool) string {
+	if semVerOnly {
+		return strings.Split(fullVersion, "-")[0]
+	}
+	return strings.Split(fullVersion, "+")[0]
```

Comment:
Probably best to use this https://github.com/decred/dcrdex/blob/2b78ab8e0e81e8996744e1c4d2d1254cc3c2efb0/dex/version/version.go#L48